src/components/Score.js

```diff
@@ -1,38 +1,49 @@
 import PropTypes from "prop-types";
 import Stack from "@mui/material/Stack";
+import Tooltip from "@mui/material/Tooltip";
 import Typography from "@mui/material/Typography";
 import ErrorIcon from "@mui/icons-material/Error";
-import qConfig from "../config/questionnaire_config";
+import { isNumber } from "../util/util";
 
 export default function Scoring(props) {
-  const { instrumentId, score, justifyContent, alignItems, scoreParams } =
-    props;
-  const getFailedScoringByInstrumentId = (instrumentId) => {
-    // TODO pass in patient education level to accurately determine what the fail score is
-    return qConfig[instrumentId] && qConfig[instrumentId].failedScores
-      ? qConfig[instrumentId].failedScores(scoreParams)
-      : [];
-  };
-  const arrFailedScores = getFailedScoringByInstrumentId(instrumentId);
+  const { score, justifyContent, alignItems, scoreParams } = props;
+  const scoreSeverity =
+    scoreParams && scoreParams.scoreSeverity
+      ? String(scoreParams.scoreSeverity).toLowerCase()
+      : null;
+  const arrAlertSeverityLevels = ["high", "moderately high", "moderate"];
```

Comment:
Does `low` need to be in this array? I saw some references in the CQL to `low`, but wasn't sure if it was the same

Reply:
thanks @ivan-c for catching this.  I have fixed so the code will only display alert icon for score that has severity level of `high` for now.  I renamed the variable and comment for clarity.  For now, the code will not display alert for severity of `moderate` / `moderately high` until specified by the project team.